config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -1320,7 +1320,7 @@ public HttpSecurity authorizeRequests(
 	 * &#064;Configuration
```

Comment:
Will you please update the copyright year on all files that you update? So, this one would change from `2002-2020` to `2002-2021`.